### PR TITLE
Add YAML file for RTD

### PR DESCRIPTION
**Pull request by JoelPasvolsky**

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+sphinx:
+    builder: html
+    configuration: docs/conf.py
+
+python:
+   version: 3.7
+   install:
+      - requirements: docs/requirements.txt
+      - method: pip
+        path: .
```
Comment (randomir) [inline .readthedocs.yml:15]: I'd invert those two, as docs deps naturally extend package deps.

**Review by randomir — APPROVED** [review]
LGTM!

File: .readthedocs.yml (hunk 1 above)
Comment (JoelPasvolsky) [reply .readthedocs.yml]: Done, thanks!

File: .readthedocs.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+sphinx:
+    builder: html
+    configuration: docs/conf.py
+
+python:
+   version: 3.7
+   install:
+      - method: pip
+      - requirements: docs/requirements.txt
+        path: .
```
Comment (randomir) [inline .readthedocs.yml:15]: Syntax error. RtD build should fail on this. `path` has to go together with `method`.

**Review by arcondello — APPROVED** [review]

**Pull request merged**